sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -124,11 +125,6 @@
      */
     private DefaultAppCenterFuture<SignInResult> mPendingSignInFuture;
 
-    /**
-     * Instance of {@link AuthTokenStorage} to store token information.
-     */
-    private AuthTokenStorage mTokenStorage;
```

Comment:
I kinda liked having a filed instead of having to use getInstance() all the time. Any reason for not having a field anymore?

Reply:
It's used inside a context now, not directly